modular_skyrat/modules/altborgs/code/robot.dm

```diff
@@ -104,7 +104,7 @@
  */
 /mob/living/silicon/robot/proc/can_rest()
 	if(model && model.model_features && ((R_TRAIT_WIDE in model.model_features) || (R_TRAIT_TALL in model.model_features)))
-		if(TRAIT_IMMOBILIZED in status_traits)
+		if(TRAIT_IMMOBILIZED in status_traits || buckled)
```

Comment:
being buckled should give this trait, any reason why it doesnt?

Reply:
was untested :p